tests/integration_tests/functional/test_cpu_template_helper.py

```diff
@@ -343,6 +343,13 @@ def test_guest_cpu_config_change(test_microvm_with_api, tmp_path, cpu_template_h
     Verify that the guest CPU config has not changed since the baseline
     fingerprint was gathered.
     """
+    if (
+        global_props.host_linux_version == "4.14"
+        and global_props.instance == "c7g.metal"
+    ):
+        # The non-SVE kernel has a different value in 0x6030000000100040 because
+        # it's an old kernel.
+        pytest.skip("old kernel has different fingerprint")
```

Comment:
It seems that this part is not unrelated to "new MSR baselines" and should be in a different commit.

Reply:
done.

Reply:
Good catch!